common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -1298,3 +1298,32 @@ def test_lms_language_change(self):
             get_selected_option_text(language_selector),
             u'English'
         )
+
+
+@attr('shard_1')
```

Comment:
Ah, sorry I wasn't super clear.  I meant to use `@attr('a11y')` instead of `@attr('shard_1')`.  It will run in the regular bok choy tests too if shard_1 is specified.